src/Amadeus/Client/Util/MsgBodyExtractor.php

```diff
@@ -38,7 +38,9 @@ class MsgBodyExtractor
      */
     public function extract($soapResponse)
     {
-        $messageBody = null;
+        if ($soapResponse === null) {
```

Comment:
@alexandr-sibov can we update PHPDoc for this method?

It currently says `@param string $soapResponse`, but it should be `string|null`.